### ISSUE TITLE
File Upload from URL

### ISSUE THREAD

**Issue by gedankenstuecke** [openSNP/snpr#249]
So, the guy who wanted to upload his full genome VCF still is having trouble doing so. But he has it available via a Dropbox link. Which made me think: Maybe we should add "upload" support from URL and we then just grab the file from there?

**Comment by gedankenstuecke** [comment]
In principle it would be [straight-forward using paperclip](https://gist.github.com/jgv/1502777) but I think we would need to include this in a Worker, because I guess otherwise the server will wait for the upload from remote-url <-> openSNP?

**Comment by tsujigiri** [comment]
Definitely. But does it actually bring any benefit?

**Comment by gedankenstuecke** [comment]
I think for larger files it will bring a benefit because especially for genome/exome VCFs the files usually are already downloadable on some server, so making people first download them and reupload them is a pain for them.

**Comment by tsujigiri** [comment]
And they wouldn't have to keep the browser open, hoping the upload goes through. Alright. Should be easy enough to do.

**Comment by gedankenstuecke** [comment]
Yes, @Treblesteph and I had some problems doing just that during the Mozilla workshop this weekend :smile:
